.github/ISSUE_TEMPLATE/translation_issue.yml

```diff
@@ -0,0 +1,81 @@
+name: "🌐 Localization/Translation issue"
+description: Report incorrect translations.
+title: ''
+labels: 
+- Issue-Bug
+- Area-Localization
+- Issue-Translation
+- Triage-Needed 
+assignees: ''
+issue_body: true
+body:
+- type: markdown
+  attributes:
+    value: |
+      Please make sure to [search for existing issues](https://github.com/microsoft/PowerTOys/issues) before filing a new one!
+- type: input
+  attributes:
+    label: Microsoft PowerToys version
+    placeholder: "0.33.1"
+    description: |
+      Hover over system tray icon or look at Settings
+  validations:
+    required: true
+- type: dropdown
+  attributes:
+    label: Utility with issue?
```

Comment:
This sentence is really strange. The word "issue" mens something different (on GitHub).
Suggestion: "Select relevant product" ?

Reply:
Or "Select affected utility:"

And then we also need to change that in the bug report template